modules/web-console/frontend/app/components/web-console-header/template.pug

```diff
@@ -14,23 +14,18 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 
-.wch-notification(ng-show='$ctrl.UserNotifications.isShown && $ctrl.UserNotifications.message' ng-bind-html='$ctrl.UserNotifications.message')
+button.web-console-header__togle-menu-button(
+    type='button'
+    title='Toggle menu'
+    ng-click='$ctrl.toggleSidebar()'
+    ng-class='::{"web-console-header__togle-menu-button-hidden": $ctrl.hideMenuButton}'
+)
+    svg(ignite-icon='menu')
 
-.wch-notification(ng-show='$root.user.becomeUsed')
-    | You are currently viewing user #[strong {{$root.user.firstName}} {{$root.user.lastName}}] as administrator. #[a(ng-click='$root.revertIdentity()') Revert to your identity?]
+a(ui-sref='default-state' ng-if='$ctrl.isAuthorized()').wch-logo-anchor
+    img.wch-logo(src='/images/ignite-logo.svg')
 
-.wch-notification.wch-notification--demo(ng-if='$root.IgniteDemoMode')
-    div(ng-controller='demoController')
-        | You are now in #[b Demo Mode]. #[a(ng-click='closeDemo();') Close Demo?]
+a(ui-sref='landing' ng-if='!$ctrl.isAuthorized()').wch-logo-anchor
```

Comment:
It is possible to replace duplicate in template with ternary operator in ui-sref?